vita3k/app/src/app_init.cpp

```diff
@@ -160,9 +159,12 @@ void init_paths(Root &root_paths) {
             root_paths.set_static_assets_path(fs::path(XDG_DATA_HOME) / app_name / dir_sep);
     }
 
+#if defined(WE_ARE_APPIMAGE)
+    auto APPDIR = getenv("APPDIR"); // Used in AppImage
     if (APPDIR != NULL) {
```

Comment:
```suggestion
    if (APPDIR != NULL && fs::exists(fs::path(APPDIR) / "usr/share/Vita3K")) {
```
much better solution